### PR TITLE
Use ecto uuid and remove standalone uuid dependency

### PR DESCRIPTION
**Pull request by waranlogesh**
- Package name of `uuid` has been renamed to `elixir_uuid`. Ref: https://github.com/zyro/elixir-uuid/issues/54.
- This fixes compilation error when this library is used in project with `:elixir_uuid` in deps.

### CODE REVIEW

**Comment by yuriploc** [comment]
I'd suggest using https://github.com/bitwalker/uniq as it's better maintained and already support the uuid v6 and v7 proposals.

File: mix.lock [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "decimal": {:hex, :decimal, "2.0.0", "a78296e617b0f5dd4c6caf57c714431347912ffb1d0842e998e9792b5642d697", [:mix], [], "hexpm", "34666e9c55dea81013e77d9d87370fe6cb6291d1ef32f46a1600230b1d44f577"},
   "earmark_parser": {:hex, :earmark_parser, "1.4.31", "a93921cdc6b9b869f519213d5bc79d9e218ba768d7270d46fdcf1c01bacff9e2", [:mix], [], "hexpm", "317d367ee0335ef037a87e46c91a2269fef6306413f731e8ec11fc45a7efd059"},
   "ecto": {:hex, :ecto, "3.9.4", "3ee68e25dbe0c36f980f1ba5dd41ee0d3eb0873bccae8aeaf1a2647242bffa35", [:mix], [{:decimal, "~> 1.6 or ~> 2.0", [hex: :decimal, repo: "hexpm", optional: false]}, {:jason, "~> 1.0", [hex: :jason, repo: "hexpm", optional: true]}, {:telemetry, "~> 0.4 or ~> 1.0", [hex: :telemetry, repo: "hexpm", optional: false]}], "hexpm", "de5f988c142a3aa4ec18b85a4ec34a2390b65b24f02385c1144252ff6ff8ee75"},
+  "elixir_uuid": {:hex, :elixir_uuid, "1.2.1", "dce506597acb7e6b0daeaff52ff6a9043f5919a4c3315abb4143f0b00378c097", [:mix], [], "hexpm", "f7eba2ea6c3555cea09706492716b0d87397b88946e6380898c2889d68585752"},
```
Comment (hammeraj) [inline mix.lock:6]: the old uuid lib is still listed in the lock file, you need to remove it with
`mix deps.clean --unused --unlock`

**Comment by waranlogesh** [comment]
> I'd suggest using https://github.com/bitwalker/uniq as it's better maintained and already support the uuid v6 and v7 proposals.

Ya existing once seems to be abandoned. What do you say @hammeraj?

**Comment by hammeraj** [comment]
> > I'd suggest using https://github.com/bitwalker/uniq as it's better maintained and already support the uuid v6 and v7 proposals.
> 
> Ya existing once seems to be abandoned. What do you say @hammeraj?

TBH for this lib we probably don't need either one. Because we are pulling in Ecto for embedded schemas we could leverage the uuid4 already present there with `Ecto.UUID.generate()`. It's only used to create a request ID for WS discovery so it's not hard to swap out.

**Review by hammeraj — APPROVED** [review]

**Pull request merged**